src/pages/Landing/Landing.css

```diff
@@ -2,14 +2,15 @@
   --button-size: 3rem;
   --button-border-radius: calc(var(--button-size) / 5);
 
-  position: fixed;
-  top: 0;
-  right: 0;
-  bottom: 0;
-  left: 0;
+  display: grid;
+  grid-template-rows: 1fr auto 1fr 3rem;
+  height: 100vh;
+  background-size: cover;
+  /* background-position: center; */
 
   .Form {
-    margin: 30vh auto;
+    -ms-grid-row: 2;
```

Comment:
Is this in use?

Reply:
Yes, it is. Grid prefixes are [disabled by default](https://github.com/postcss/autoprefixer/issues/817) in Autoprefixer.

Reply:
Just noticed that Edge 16 support the new specification. I'll remove this.